internal/ingress/store/store.go

```diff
@@ -109,12 +110,22 @@ type CacheStores struct {
 
 // New creates a new object store to be used in the ingress controller
 func New(cs CacheStores, ingressClass string, skipClasslessIngress bool, logger logrus.FieldLogger) Storer {
+	var ingressClassHandling annotations.ClassHandling
+	// TODO this is a placeholder for the eventual consumer flag
+	// for now it hard-codes the default
+	kongConsumerClassHandling := annotations.ExactOrEmptyClassMatch
+	if skipClasslessIngress {
```

Comment:
this `bool` looks like it could be fully obsoleted and replaced with `ClassHandling`, no?

Reply:
Don't think so. These bools (just the one now, but two in the future) are derived from flags, where we want users to specify a binary choice--i.e. we don't want to allow them to specify the handling directly, since they'd be able to choose "use ignore handling for Ingress objects".